### ISSUE TITLE
AndroidRuntime NullPointerException

### ISSUE THREAD

**Issue by IgorGanapolsky** [sockeqwe/mosby#29]
Hello,
when using your **MvpViewStateFragment** in my app and rotating the device, I get this crash:
>java.lang.NullPointerException: ViewState is null! That's not allowed
            at com.hannesdorfmann.mosby.mvp.viewstate.ViewStateManager.saveInstanceState(ViewStateManager.java:103)
            at com.hannesdorfmann.mosby.mvp.viewstate.MvpViewStateFragment.saveViewStateInstanceState(MvpViewStateFragment.java:79)
            at com.hannesdorfmann.mosby.mvp.viewstate.MvpViewStateFragment.onSaveInstanceState(MvpViewStateFragment.java:70)
            at android.support.v4.app.Fragment.performSaveInstanceState(Fragment.java:1936)
            at android.support.v4.app.FragmentManagerImpl.saveFragmentBasicState(FragmentManager.java:1654)
            at android.support.v4.app.FragmentManagerImpl.saveAllState(FragmentManager.java:1722)
            at android.support.v4.app.FragmentActivity.onSaveInstanceState(FragmentActivity.java:527)
            at com.hannesdorfmann.mosby.MosbyActivity.onSaveInstanceState(MosbyActivity.java:44)
    

Please advise,
Igor

**Comment by GNima** [comment]
Probably same as #26

**Comment by sockeqwe** [comment]
@IgorGanapolsky are you putting the fragments on the back stack? if yes, there was an issue with the backstack in Version 1.0.0. This should be fixed in the upcoming release 1.1.0. Could you please test your app with`1.1.0-SNAPSHOT` ? I guess @GNima  is right and this issues is related to the backstack problem in `1.0.0`

```groovy
allprojects {
  repositories {
    jcenter()
    maven{
      url 'https://oss.sonatype.org/content/repositories/snapshots/'
    }
}
```

Version is `1.1.0-SNAPSHOT`

Please let me know if the SNAPSHOT Version solves that problem.
